openshift/manage

```diff
@@ -253,7 +261,7 @@ deployApp(){
 
     # Deploy or update the configuration ...
     updateDeploymentParams ${DEPLOYMENT_ENV_NAME}
-    if dcExists ${_appName}; then
+    if dcExists "allure-ui"; then
```

Comment:
For `allure` there is no `dc` named as the app label, so I picked one of the two components to check whether the app was already deployed.